src/libs/deprecatedAPI.js

```diff
@@ -75,31 +75,6 @@ function Get(parameters, shouldUseSecure = false) {
     return Network.post(commandName, parameters, CONST.NETWORK.METHOD.POST, shouldUseSecure);
 }
 
-/**
- * @param {Object} parameters
- * @param {Number} parameters.reportID
- * @param {String} parameters.paymentMethodType
- * @param {Object} [parameters.newIOUReportDetails]
- * @returns {Promise}
- */
-function PayIOU(parameters) {
-    const commandName = 'PayIOU';
-    requireParameters(['reportID', 'paymentMethodType'], parameters, commandName);
-    return Network.post(commandName, parameters);
-}
-
-/**
- * @param {Object} parameters
- * @param {Number} parameters.reportID
- * @param {Object} [parameters.newIOUReportDetails]
- * @returns {Promise}
- */
-function PayWithWallet(parameters) {
-    const commandName = 'PayWithWallet';
-    requireParameters(['reportID'], parameters, commandName);
-    return Network.post(commandName, parameters);
-}
-
```

Comment:
@youssef-lr Correct me if I am wrong, but I think this will diable the PayWithWallet flow, we still need ot keep that in, the clean up should be done in the PRs by @sketchydroide 

cc @luacmartins

Reply:
Ah nice catch, yeah this should be removed in @sketchydroide's PR. Will update.